src/ImageSharp/Formats/Jpeg/Components/Block8x8.cs

```diff
@@ -271,12 +267,7 @@ public void TransposeInplace()
             // row #6
             Swap(ref Unsafe.Add(ref elemRef, 55), ref Unsafe.Add(ref elemRef, 62));
 
-            static void Swap(ref short a, ref short b)
-            {
-                short tmp = a;
-                a = b;
-                b = tmp;
-            }
+            static void Swap(ref short a, ref short b) => (b, a) = (a, b);
```

Comment:
Just as info:
Resulting machine code will be the same, but the IL-size is with the tuple-based approach a bit larger than with the local (old version).

I don't think it matters here, but potential drawbacks
* JIT may give up inlining if any budget may exceed earlier --> add `MethodImpl.AggressiveInlining` (or the ImageSharp-equivalent) to be on the safe side?
* on products where IL size matters this could have a negative effect (TBH I don't know if two bytes more or less count here, as the reduction overall is much bigger that this PR does, but just to point it out)

Reply:
Another poor analyser suggestion 😔

Reply:
I'm no compiler engineer*, but for me this pattern is easy to recognize, so "optimal" code should be emitted, even if Roslyn isn't a optimizing compiler (that's the role of the JIT), but regarding IL-size optimizations count towards Roslyn.

\* so I have somewhat limited understanding on how things work there